examples/dotnet-diagnostics/DiagnosticsExample/MainWindow.xaml.cs

```diff
@@ -199,6 +199,29 @@ await Task.Run(async () =>
         });
     }
 
+    private async void FindIntentButton_Click(object sender, RoutedEventArgs e)
+    {
+        await Task.Run(async () =>
+        {
+            Dispatcher.Invoke(() =>
+            {
+                DiagnosticsText += "\nFinding intent for ViewChart...";
```

Comment:
It would probably make sense to reset the text at the beginning of testing new functionality